task2/src/index.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+
+import userRouter from './routes/user';
+import usersRouter from './routes/users';
+
+const app = express();
+const port = process.env.PORT || 3001;
+
+app.listen(port);
+app.use(express.json());
+app.use('/user', userRouter);
```

Comment:
Usuall CRUD operations for single entity like user can be created like this:
GET /users/:id - get a user
POST /users - create a user
PUT /users/:id - update a user
DELETE /users/:id - delete a user

For autosuggestion you could use something like /users/autoSuggest or at least /autoSuggest

Reply:
Changed to following way

GET /users/:id - get a user
POST /users - create a user
PUT /users/- update a user
DELETE /users/:id - delete a user

/getAutoSuggestUsers